### PR TITLE
New Blueprints + Blueprint Updates

### PR DESCRIPTION
**Pull request by blakeburch**
- Includes Blueprint Library pages for:
    - Mode
    - Census
    - Rudderstack
    - Databricks
    - Hightouch

- Updates all Blueprint Library pages with YAML definitions and any changed variables.
- Adds blueprint updates to the changelog
- Adds new vendors to overview list
- Upgrades to 2.0.1

### CODE REVIEW

File: docs/blueprint-library/amazon-athena/amazon-athena-execute-query.md [hunk 1]
```diff
@@ -33,3 +33,26 @@ Execute any SQL query against Amazon Athena. Perfect for creating multi-step SQL
...
+    Access Key ID: null ## REQUIRED
+    Secret Access Key: null ## REQUIRED
+    Region: us-east-2 ## REQUIRED
+  type: BLUEPRINT
+guardrails:
+  retry_count: 1
+  retry_wait: 0s
+  runtime_cutoff: 4h0m0s
+  exclude_exit_code_ranges:
+    - "0"
```
Comment (katthartic) [inline docs/blueprint-library/amazon-athena/amazon-athena-execute-query.md:57]: It looks like it is filling in `0` as the default exit code range again.

File: docs/blueprint-library/amazon-athena/amazon-athena-execute-query.md [hunk 2]
```diff
@@ -33,3 +33,26 @@ Execute any SQL query against Amazon Athena. Perfect for creating multi-step SQL
...
 
+## YAML
+
+Below is the YAML template for this Blueprint and can be used in the Fleet [YAML Editor](../../reference/fleets/yaml-editor.md).
+
+```yaml
+source:
+  blueprint: Amazon Athena - Execute Query
+  inputs:
+    Database: null ## REQUIRED
```
Comment (katthartic) [inline docs/blueprint-library/amazon-athena/amazon-athena-execute-query.md:44]: There was the bug with null being interpreted as a string... is that still the case?

File: docs/blueprint-library/google-bigquery/google-bigquery-store-query-results-in-google-cloud-storage.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ keywords:
 
 Turn the results of your SQL SELECT statement into CSV files that get stored in Google Cloud Storage (GCS).
 
-When exporting data from BigQuery, a [file cannot contain more than 1GB of data](https://cloud.google.com/bigquery/docs/exporting-data). As a result, this Blueprint automatically splits up data that is &gt;1GB in size into multiple files. The name of these files will be the Bucket File Name provided, with `_#` base 0 enumeration appended to the file name, before the file extension. This is the [default behavior provided by Google](https://cloud.google.com/bigquery/docs/exporting-data#exporting_data_into_one_or_more_files).
+When exporting data from Bigquery, a [file cannot contain more than 1GB of data](https://cloud.google.com/bigquery/docs/exporting-data). As a result, this Blueprint automatically splits up data that is &gt;1GB in size into multiple files. The name of these files will be the Bucket File Name provided, with `_#` base 0 enumeration appended to the file name, before the file extension. This is the [default behavior provided by Google](https://cloud.google.com/bigquery/docs/exporting-data#exporting_data_into_one_or_more_files).
```
Comment (ericelsken) [inline docs/blueprint-library/google-bigquery/google-bigquery-store-query-results-in-google-cloud-storage.md:21]: Is this case change correct? It looks stylized differently on the web.

File: docs/blueprint-library/hightouch/hightouch-authorization.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+keywords:
+  - hightouch
+  - blueprint
+  - template
+  - authorization
+---
+
+# Hightouch Authorization
+
+To get started with any Hightouch Blueprints, you will need to create an API Key. This API Key is a secure way for Shipyard to access Hightouch and will be required for **every** Census Blueprint. Additionally, you'll need a Sync ID.
```
Comment (ericelsken) [inline docs/blueprint-library/hightouch/hightouch-authorization.md:16]: Census -> Hightouch

File: docs/blueprint-library/hightouch/hightouch-check-sync-status.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    - template
+---
+
+# Hightouch - Check Sync Status
+
+## Overview
+
+&gt; ## **First time using this Blueprint? Make sure you follow our [Hightouch authorization guide](https://www.shipyardapp.com/docs/blueprint-library/hightouch/hightouch-authorization/)**.
+
+Check for the status of a Census sync. The Vessel&#39;s exit code and status will match the status of the latest sync.
```
Comment (ericelsken) [inline docs/blueprint-library/hightouch/hightouch-check-sync-status.md:19]: Census -> Hightouch

File: docs/blueprint-library/census/census-trigger-sync.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+&gt; ## **First time using this Blueprint? Make sure you follow our [Census authorization guide](https://www.shipyardapp.com/docs/blueprint-library/census/cesnsus-authorization/)**.
+
+Immediately execute a Census snyc. The sync must already be set up within the Census interface.
+
+If a sync is already running for the specified connector, it will be stopped and restarted.
+
+This Blueprint will only kick off the sync and will almost always return a status of success. It will not wait around to verify if the sync was successfully completed, but it will create and store the source ID used to `shipyard-artifacts/census-blueprints/variables/sync_run_id.pickle`
+
+**Recommended Setup:**
+1. This Vessel should be immediately followed by a Vessel built from the Check Sync Status Blueprint so you can verify the status and completion of your sync.
```
Comment (katthartic) [inline docs/blueprint-library/census/census-trigger-sync.md:26]: Should `Check Sync Status` be italicized or something?

**Review by ericelsken — CHANGES_REQUESTED** [review]

File: docs/blueprint-library/email/email-send-message-with-file.md [hunk 7]
```diff
@@ -18,6 +18,10 @@ keywords:
 
 Send a message to anyone in the world using email, attaching one or more files.
 
+To upload multiple files, use [Regex Match Type](https://www.shipyardapp.com/docs/reference/blueprint-library/match-type/).
+
+Due to email file size constraints, when the the total file size &gt;10MB, Shipyard will automatically zip the files under the name &#34;Archive.zip&#34;.
```
Comment (katthartic) [inline docs/blueprint-library/email/email-send-message-with-file.md:23]: Same here with the `&#34`.

File: docs/blueprint-library/google-bigquery/google-bigquery-store-query-results-in-google-cloud-storage.md (hunk 3 above)
Comment (katthartic) [inline docs/blueprint-library/google-bigquery/google-bigquery-store-query-results-in-google-cloud-storage.md]: It is supposed to be `BigQuery` with a capital B, I thought.

File: docs/blueprint-library/mode/mode-authorization.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+4. Provide a descriptive New token name like "Shipyard".
+5. Click **Create Token**.
+6. Copy both the Token and Token Password and store them somewhere safe. These values will be used for every Mode Blueprint.
+
+![Create Mode Access Token](../../.gitbook/assets/shipyard_2022_08_16_17_28_00.png)
+
+## Finding your Workspace Name
+On any page of Mode, the workspace name can be found in the URL.
+
+For example, if your're on the main dashboard and the URL is `https://app.mode.com/my-workspace/spaces/90210abcdef` then your workspace name would be `my-workspace`.
```
Comment (katthartic) [inline docs/blueprint-library/mode/mode-authorization.md:37]: `your're` 👉 `you're`

**Review by ericelsken — APPROVED** [review]

**Comment by katthartic** [comment]
@blakeburch FYI... I'm still reviewing but have found stuff as well.

**Review by katthartic — CHANGES_REQUESTED** [review]

File: docs/blueprint-library/amazon-athena/amazon-athena-execute-query.md (hunk 1 above)
Comment (blakeburch) [reply docs/blueprint-library/amazon-athena/amazon-athena-execute-query.md:57]: Need to have @forstmeier fix that. Manually updated for now.

File: docs/blueprint-library/amazon-athena/amazon-athena-execute-query.md (hunk 2 above)
Comment (blakeburch) [reply docs/blueprint-library/amazon-athena/amazon-athena-execute-query.md:44]: Not that I'm aware of. nulls were find for WtW

File: docs/blueprint-library/census/census-trigger-sync.md (hunk 6 above)
Comment (blakeburch) [reply docs/blueprint-library/census/census-trigger-sync.md:26]: No official rule on this. I would rather let future S&I make that call and ensure consistency.

**Comment by blakeburch** [comment]
@katthartic Just added your feedback. That dismissed @ericelsken’s review. Can someone give it the final stamp?

**Review by ericelsken — APPROVED** [review]

**Review by ericelsken — APPROVED** [review]

**Review by katthartic — APPROVED** [review]
Don't think this matters but in some places you have `exclude_exit_code_ranges:` with nothing underneath and in other places, it is missing entirely.

**Pull request merged**